Dockerfile

```diff
@@ -1,19 +1,32 @@
-FROM python:3.9-buster AS base
+FROM python:3.9.2-slim-buster AS base
 RUN mkdir /app
 COPY *.toml /app/
 WORKDIR /app
 RUN pip3 install poetry \
-    && poetry config virtualenvs.create false \
-    && poetry install --no-dev
+    && poetry config virtualenvs.create false --local \
+    && poetry install --no-dev \
+    && apt-get update && apt-get install -y curl
 
 FROM base AS dev
 WORKDIR /app
 EXPOSE 5000
 CMD ["poetry", "run", "flask", "run", "-h", "0.0.0.0"]
 
 FROM base AS prod
-ENV FLASK_ENV=production
 COPY . /app
 WORKDIR /app
-EXPOSE 8000
-CMD ["poetry", "run", "gunicorn", "-b", "0.0.0.0", "todo_app.app:create_app()"]
\ No newline at end of file
+EXPOSE 5000
+ENV PORT=${PORT}
```

Comment:
This is trying to set the PORT environment variable as the existing PORT environment variable - it has no effect!
What we could do is set a default value here, e.g. 
`ENV PORT=8000`
If an alternative is provided at runtime (e.g. via `-e PORT=6000` ) then the alternative value will still be used. This would fix up the current issue that your README docker run command for prod is now out of date

Reply:
Adding fix to exercise-9 branch